src/src/components/Login.vue

```diff
@@ -0,0 +1,131 @@
+<template>
+  <form class="login" @submit.prevent="handleLogin">
+    <img src="../assets/images/vaultN-logo.svg" class="login-logo" />
+    <h1>{{ contentLogin.loginHeadlineText }}</h1>
+    <CustomInput :input="customEmailInput" @valueChanged="handleTextChanged"/>
+    <CustomInput :input="customPasswordInput" @valueChanged="handleTextChanged"/>
+    <FormErrors v-if="errroMessages" :messages="errroMessages"></FormErrors>
+    <a href="#/forgot" class="link">{{ contentLogin.loginForgotPasswordText }}</a>
+    <CustomButton :button="customLoginButton" @buttonClicked="handleLogin"/>
+    <!-- <button class="btn btn-login" type="submit">{{ contentLogin.loginButtonText }}</button> -->
```

Comment:
Saved for later?

Reply:
yes, I had to push these inbetween since this was an init branch. I'll resolve these in the next commit.